pyproject.toml

```
@@ -45,6 +45,7 @@ sphinx_rtd_theme = {version="^1.0", optional=true}
 grpcio = {version=">=1.49.0,<2.0", optional = true}
 protobuf = { version = "^4.21", optional = true }
 hightime = "^0.2.1"
+click = "^8.0.0"
```

It is not a given that click 9.0 will break compatibility with existing applications. Please use `>=` instead of `^` so that nidaqmx-python will not limit users to click 8.x when this happens.
```suggestion
click = ">=8.0.0"
```